From 97d921e384e25cf471917f544e1c6f75249e1198 Mon Sep 17 00:00:00 2001
From: ckkim <changkon12@gmail.com>
Date: Thu, 5 Mar 2020 14:26:42 +0900
Subject: [PATCH 053/109] ODROID-XU4: arm/dts: remove write-protect pin from SD
 card

This patch removes the write-protect pin from SD card since it's not
being supported by a hardware and used at all.

Signed-off-by: ckkim <changkon12@gmail.com>
Change-Id: I72af94f7fd0a01e65f7ca3e15ccd66e797395df4
---
 arch/arm/boot/dts/exynos5422-odroid-core.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/exynos5422-odroid-core.dtsi b/arch/arm/boot/dts/exynos5422-odroid-core.dtsi
index e7505f46605b..13ae1e5fd4f0 100644
--- a/arch/arm/boot/dts/exynos5422-odroid-core.dtsi
+++ b/arch/arm/boot/dts/exynos5422-odroid-core.dtsi
@@ -970,9 +970,10 @@ &mmc_2 {
 	samsung,dw-mshc-sdr-timing = <0 4>;
 	samsung,dw-mshc-ddr-timing = <0 2>;
 	pinctrl-names = "default";
-	pinctrl-0 = <&sd2_clk &sd2_cmd &sd2_cd &sd2_wp &sd2_bus1 &sd2_bus4>;
+	pinctrl-0 = <&sd2_clk &sd2_cmd &sd2_cd &sd2_bus1 &sd2_bus4>;
 	bus-width = <4>;
 	cap-sd-highspeed;
+	disable-wp;
 	max-frequency = <200000000>;
 	vmmc-supply = <&ldo19_reg>;
 	vqmmc-supply = <&ldo13_reg>;
-- 
2.25.1

